### PR TITLE
Playbook bugs

### PR DESCRIPTION
**Pull request by kylecesmat**
Fixes:
- Removed `react-router-scroll` as we weren't utilizing it and caused issues with TOC anchors
- Conditionally throw not-found state so it doesn't flash on load

### CODE REVIEW

**Comment by kylecesmat** [comment]
cc @paulathevalley @bmathews

File: src/views/docs.js [hunk 1]
```diff
@@ -20,19 +20,33 @@ class Docs extends React.Component {
...
     }
   }
 
+  getCurrentDocs() {
+    let docsComponent = { file: "" };
+    let counter = 0;
+    config.forEach((doc) => {
+      if (doc.route === this.props.location.pathname || `${doc.route}/` === this.props.location.pathname) {
+        docsComponent = doc;
+        return false;
```
Comment (bmathews) [inline src/views/docs.js]: Returning false in a `.forEach` doesn't stop the iteration. `.every` does though!

File: src/views/docs.js [hunk 2]
```diff
@@ -20,19 +20,33 @@ class Docs extends React.Component {
...
+      }
+      counter++;
+    });
+
+    this.setState({
+      docs: docsComponent,
+      loadingError: counter === config.length
+    });
```
Comment (bmathews) [inline src/views/docs.js]: `setState` in `componentDidUpdate` will cause a double render, and `componentDidUpdate` isn't called during the initial render. Maybe `componentWillReceiveProps` would work better here?

Comment (kylecesmat) [inline src/views/docs.js]: @bmathews not if I have it inside of the if statement though, right? I mean I could use `componentWillReceiveProps` for this & the window scroll, would that be better?

File: src/views/docs.js (hunk 1 above)
Comment (kylecesmat) [inline src/views/docs.js]: Oh nice. It does make sense now looking at it that the return doesn't do anything :P

**Comment by kylecesmat** [comment]
@bmathews could use a re-review!

**Comment by bmathews** [comment]
:+1:

**Pull request merged**